version.mk

```diff
@@ -46,6 +48,13 @@ endif
 ifeq ($(GCC), true)
 	@(if ! [ $(GCC_REQ) > /dev/null 2>&1 ]; then echo "\e[1;31mWARNING: You seem not having \"gcc\" installed\e[1;m" && exit 1 ; fi)
 endif
+ifeq ($(DBMLCLI), true)
+	@(if ! [ $(DBMLCLI) > /dev/null 2>&1 ]; then echo "\e[1;31mWARNING: You seem not having \"dbmlcli\" installed\e[1;m" && exit 1 ; fi)
```

Comment:
This and the below check are wrong, we should validate that the command is actually there in the system, invoking it. And if possible check also the tool version.